.github/TD.HTML

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>

```

Comment:
I recommend changing your html to code to use the onclick event. The onclick event occurs when the user clicks on an element.
`<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>counter</title>
</head>
<link rel="stylesheet" href="TD.css">
<body>
    <div class="coun">counter
    </div>
    <div class="number">0</div>
    <div class="button">
    <button class="btns decrease" onclick="handleClick(event)">
        decrease
    </button>
    <button class="btns reset" onclick="handleClick(event)">
        reset
    </button>
    <button class="btns increase" onclick="handleClick(event)">
        increase
    </button>
    </div>
    <script src="TD.js"></script>
</body>
</html>`

Here I'm using the onlick event to call a functions which I've defined in JavaScript and I'm passing an event as a parameter.